plugins/techdocs-node/src/stages/publish/local.ts

```diff
@@ -199,28 +199,28 @@ export class LocalPublish implements PublisherBase {
       }
 
       // Generate a lower-case entity triplet path.
-      const [_, namespace, kind, name, ...rest] = req.path.split('/');
+      const [prefix, namespace, kind, name, ...rest] = req.path.split(path.sep);
 
       // Ignore non-triplet objects.
       if (!namespace || !kind || !name) {
         return next();
       }
 
       const newPath = [
-        _,
+        prefix,
         namespace.toLowerCase(),
         kind.toLowerCase(),
         name.toLowerCase(),
         ...rest,
-      ].join('/');
+      ].join(path.posix.sep);
 
       // If there was no change, then let express.static() handle the request.
       if (newPath === req.path) {
         return next();
       }
 
       // Otherwise, redirect to the new path.
-      return res.redirect(req.baseUrl + newPath, 301);
+      return res.redirect(301, req.baseUrl + newPath);
```

Comment:
This is just fixed because deprecation warning

`express deprecated res.redirect(url, status): Use res.redirect(status, url) instead at src\stages\publish\local.ts:223:18`